src/main/java/uk/gov/digital/ho/hocs/workflow/BpmnService.java

```diff
@@ -37,17 +40,19 @@
     private final CamundaClient camundaClient;
     private final InfoClient infoClient;
     private final Clock clock;
-
+    private final WorkflowService workflowService;
 
     @Autowired
     public BpmnService(CaseworkClient caseworkClient,
                        CamundaClient camundaClient,
                        InfoClient infoClient,
-                       Clock clock) {
+                       Clock clock,
+                       WorkflowService workflowService) {
+        this.clock = clock;
         this.caseworkClient = caseworkClient;
         this.camundaClient = camundaClient;
         this.infoClient = infoClient;
-        this.clock = clock;
```

Comment:
clock is passed in but no longer assigned?

Reply:
It is assigned to this.clock on line 52 - I must have moved it when resolving conflicts in this area when I rebased

Reply:
Oh yeah sorry not sure how I didn't see it there!